tests/unit/lib/Any_unittest.cc

```diff
@@ -691,3 +691,65 @@ TEST(Any, lazy_cloneable_visitable)
 	EXPECT_FALSE(any.visitable());
 	EXPECT_THROW(any.visit(nullptr), std::logic_error);
 }
+
+TEST(AnyParameterProperties, old_api_default)
+{
+	AnyParameterProperties params = AnyParameterProperties();
+
+	EXPECT_EQ(
+	    params.get_model_selection(),
+	    EModelSelectionAvailability::MS_NOT_AVAILABLE);
+	EXPECT_EQ(
+	    params.get_gradient(), EGradientAvailability::GRADIENT_NOT_AVAILABLE);
+	EXPECT_FALSE(params.get_model());
+}
+
+TEST(AnyParameterProperties, new_api_default)
+{
+	AnyParameterProperties params = AnyParameterProperties();
+
+	EXPECT_TRUE(params.equals(ParameterProperties()));
+	EXPECT_FALSE(params.has_property(ParameterProperties::MODEL));
```

Comment:
as said before, can we not explicitly test that those are not set, but in fact that *none* are set. Otherwise we will have to modify tests when we add new attributes

Reply:
I did think I saw that comment in the previous PR, but the next day the comment disappeared..

Reply:
Yeah that happens on force push. I keep the notification emails to not loose comments when github hides them.